pkg/R/DataFrame.R

```diff
@@ -768,6 +798,20 @@ setMethod("select", signature(x = "DataFrame", col = "Column"),
             dataFrame(sdf)
           })
 
+setMethod("select",
+          signature(x = "DataFrame", col = "list"),
```

Comment:
I see -- yes i think it is a good feature to have. I wrote up a few more APIs we should support for column selection at [1]
and one of them was to include a way to specify a list of column names.

[1] https://sparkr.atlassian.net/browse/SPARKR-189?focusedCommentId=12320&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-12320